.github/workflows/ci.yml

```
@@ -108,6 +108,35 @@ jobs:
           make setup
           make test # This target should handle setup, build, install, and pytest
 
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)